component/config.jsonnet

```diff
@@ -70,6 +70,46 @@ local config = [
             hs.status = "Progressing"
             hs.message = "Waiting for provider to be installed"
             return hs
+        operators.coreos.com/Subscription:
+          health.lua.useOpenLibs: true
+          health.lua: |
+            -- Base check copied from upstream
```

Comment:
Nitpick: Adding a link to the upstream check and/or more clearly marking the difference to upstream might be helpful. Inline without any syntax highlighting it's IMO hard to see at a first glance what was added.

Reply:
done in latest version